EasyClangComplete.py

```diff
@@ -237,6 +272,33 @@ def completion_finished(self, future):
             # are new completions to show there. Otherwise let it be.
             SublBridge.show_auto_complete(active_view)
 
+    def on_hover(self, view, point, hover_zone):
+        """Function that is called when mouse pointer hovers over text.
+
+        Triggers showing popup with additional information about element under
+        cursor.
+
+        """
+        if self.saving:
+            return
+        if not Tools.is_valid_view(view):
+            return
+
+        settings = self.settings_manager.settings_for_view(view)
+        if settings.show_type_info == False:
+            return
+        if hover_zone != sublime.HOVER_TEXT:
+            return
+        completion_request = tools.CompletionRequest(view, point)
```

Comment:
Again, seems to be a wrong name. Up to the point, where we might want to also rename the `CompletionRequest` class.